README.md

```diff
@@ -1,59 +1,71 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# WEB DEVELOPMENT FREE COURSE Website
 
-> One paragraph statement about the project.
+> WEB DEVELOPMENT FREE COURSES.
 
-![screenshot](./app_screenshot.png)
+  This capstone project is a website for free web development courses build based on an online website for a conference. The original design idea belongs to [Cindy Shin](https://www.behance.net/adagio07).  [Link to the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015).
 
-Additional description about the project and its features.
+This website has two pages.
 
-## Built With
+The Home page:
 
-- Major languages
-- Frameworks
-- Technologies used
+- Desktop Version 
 
-## Live Demo
+![Screenshot 2021-12-23 140814](https://user-images.githubusercontent.com/92228303/147248775-cc3a749a-1b1b-47f2-9d11-c7c2d5ca3c74.png)
 
-[Live Demo Link](https://livedemo.com)
+- Mobile version
 
+![Screenshot 2021-12-23 140904](https://user-images.githubusercontent.com/92228303/147249171-789fd99e-1bd2-495f-a843-03bfd1d64c54.png)
 
-## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+The About page: 
 
+- Desktop Version
 
-To get a local copy up and running follow these simple example steps.
+![Screenshot 2021-12-23 140954](https://user-images.githubusercontent.com/92228303/147248998-7ed7c838-9cc4-4f0e-8282-c772ac8483bd.png)
 
-### Prerequisites
+- Mobile version
 
-### Setup
+![Screenshot 2021-12-23 140935](https://user-images.githubusercontent.com/92228303/147249243-47b042fe-b138-4ff0-97e0-145e34851035.png)
 
-### Install
+Additional features:
 
-### Usage
+- The section "Teaching Team" is created dynamically in JavaScript.
 
-### Run tests
+- The interaction in mobile menu is build by JavaScript and CSS.
 
-### Deployment
+## Built With
 
+- HTML
+- CSS
+- JavaScript
 
+## Live Demo
 
-## Authors
+[Live Demo Link](https://timmychan99.github.io/WebDev_Course/)
+
```

Comment:
- [ ] [OPTIONAL] Your video is impressive :+1: . It will be nice if you could add it here in your readme for other visitors who will love to know what your project is all about.